Indexer/Indexer.php

```diff
@@ -640,8 +644,13 @@ public function getClient()
                 $this->apiSettings['application_id'],
                 $this->apiSettings['api_key']
             );
+
+            $this->client->setConnectTimeout(
+                $this->apiSettings['connection_timeout']
```

Comment:
You will get an _undefined index_ notice here if the `setApiSettings` method was called without the new `$connection_timeout` parameter.

I guess this method should be called only if `$this->apiSettings['connection_timeout']` is not set.